dockertest/subtestbase.py

```diff
@@ -168,6 +172,21 @@ def failif_not_in(needle, haystack, description=None):
                              (expect_s, description, haystack))
 
     @classmethod
+    def failif_not_redhat(cls, xcept=DockerTestNAError):
+        """
+        Raises an exception if this is a non-RHEL/RHELAH System
+
+        :param xcept: Non-default exception to raise
+        :raise DockerTestNAError: By default
+        """
+        try:
+            with open(cls.redhat_release_filepath, 'rb') as rhrel:
+                if 'Enterprise' not in rhrel.read():
+                    raise xcept("Expecting a Red Hat Enterprise Linux System")
+        except IOError:
+            cls.logwarning("Problem checking /etc/redhat-release")
```

Comment:
TBH, I'm not sure what to do in this case.  Figured I'd toss in a logging message so you'd notice and give me a better solution :grinning: 

Thinking about it more...it seems like in all cases, raising a ``TestNAError()`` (with a different message) is what it should do.  Ya?